README.md

```diff
@@ -62,3 +62,12 @@ App Reviewer partners are encouraged, but not required to, monitor the repo and
 ## Legal considerations
 
 The App Mining program is operated by Blockstack PBC and is subject to various applicable laws and regulations. Even if a process change is technically possible, it might not legally possible due to regulatory conditions. Please see [this link](https://blog.blockstack.org/introducing-app-mining/) for a list of disclosures.
+
+## Definition of high quality applications
```

Comment:
@friedger Thanks for this. Good idea.  I think I would write something a bit more specific.

> ## What is a high quality application?
> 
> A high quality application in the App Mining program performs well in these areas:
> 
> * adheres to the [Principles of Blockstack apps](https://docs.blockstack.org/develop/dapp_principles.html)
> * attracts users through innovation in both features and visual design
> * supports first-time users through good user interaction (UX) and education 
> * communicates a marketing plan and feature roadmap that attracts investors

```markdown
 ## What is a high quality application?

A high quality application in the App Mining program performs well in these areas:

* adheres to the [Principles of Blockstack apps](https://docs.blockstack.org/develop/dapp_principles.html)
* attracts users through innovation in both features and visual design
* supports first-time users through good user interaction (UX) and education 
* communicates a marketing plan and feature roadmap that attracts investors
```

Reply:
@friedger suggested changes here